### PR TITLE
Add plugin information in 'About' window

### LINKED ISSUE #268: List Plugins (RBin/Rasm/IO)
Add a button to List Plugins (RBin/Rasm/IO) in About Cutter

### PR DESCRIPTION
**Pull request by Abogical**
Fixes #268

### CODE REVIEW

**Comment by xarkes** [comment]
Hmm I don't think this should be named "Plugin" but more "RadarePlugin" or "BinAsmPlugin", I don't know. cc @Maijin 
Also the issue states that it should list RBin, Rasm and RIO plugins, while your pull request only show the RBin ones.
Can you please add the others?

**Comment by Abogical** [comment]
`oL` command lists IO plugins. What command is used to list Rasm plugins?

**Comment by xarkes** [comment]
Maybe this can be helpful:
https://github.com/radare/radare2/blob/master/binr/rasm2/rasm2.c#L73
Also there are core plugins available via `Ls`

File: src/dialogs/PluginsDialog.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#include "Cutter.h"
+#include "utils/Helpers.h"
+
+PluginsDialog::PluginsDialog(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::PluginsDialog)
+{
+    ui->setupUi(this);
+
+    for(auto plugin : Core()->getInstance()->getRBinPluginDescriptions())
```
Comment (xarkes) [inline src/dialogs/PluginsDialog.cpp]: Use `Core()->getRBinPluginDescriptions()` no need to call getInstance, it is redundant.

**Comment by Maijin** [comment]
- Rbin iLj
- RAsm e asm.arch=?j
- RIo oLj
- Rcore Lsj must be implemented in radare2 first

Always use JSON

**Comment by xarkes** [comment]
And rename from Plugins to R2Plugins

**Comment by Abogical** [comment]
Discovered a bug in `oLj`. This PR now depends on radare/radare2#9634 .

**Comment by Abogical** [comment]
@Maijin `e asm.arch=?j` doesn't produce a JSON output. However it can still easily be parsed, should I work on it's output anyway?

**Comment by Abogical** [comment]
This PR now also depends on implementation of `Lsj` in radare2/radare#9637

**Comment by xarkes** [comment]
Btw @Abogical, are you a student interesting in the GSoC? If so, please contact me on irc or Telegram, ...

**Comment by xarkes** [comment]
I think RAsm plugins is still missing, after that it will be good for merge!

**Comment by Maijin** [comment]
if a JSON output is missing, better adding it than parsing output. Can you do it or open an issue on radare2 repository ?

**Comment by Abogical** [comment]
Ok it seems `evj` already does this.

**Comment by xarkes** [comment]
`evj asm.arch` you mean?

**Comment by Abogical** [comment]
Yes

**Comment by Abogical** [comment]
Done!

**Comment by Maijin** [comment]
ok perfect then

**Pull request merged**